src/main/java/com/sap/cloudfoundry/client/facade/adapters/RawCloudApplication.java

```diff
@@ -26,12 +24,11 @@
 import com.sap.cloudfoundry.client.facade.domain.ImmutableStaging;
 import com.sap.cloudfoundry.client.facade.domain.PackageState;
 import com.sap.cloudfoundry.client.facade.domain.Staging;
-import com.sap.cloudfoundry.client.facade.util.JsonUtil;
 
 @Value.Immutable
 public abstract class RawCloudApplication extends RawCloudEntity<CloudApplication> {
 
-    public abstract Resource<ApplicationEntity> getResource();
+    public abstract Resource getResource();
```

Comment:
Done, but used applications.v3.list to get the applicationResourse because GetApplicationResponse does not extend ApplicationResource.... it extends only Application

Reply:
`ApplicationResource` isn't any different from `Application`. It extends it but doesn't add any new methods. The `get` call would probably be more efficient than `list`

Reply:
Done.
Actually, the application has to be found by its name which isn't possible with GetApplicationRequest